packages/next/src/build/webpack/loaders/next-metadata-image-loader.ts

```diff
@@ -109,6 +113,21 @@ async function nextMetadataImageLoader(this: any, content: Buffer) {
               : `${imageSize.width}x${imageSize.height}`,
         }),
   }
+  if (type === 'openGraph' || type === 'twitter') {
+    let alt
+    try {
+      const altPath = await resolveAltText(
+        context,
+        path.join(path.dirname(resourcePath), fileNameBase + '.alt.txt')
```

Comment:
Passing an absolute path to resolve is a bit weird.

When you already have an absolute path, there is no need to resolve it.